graph.h

```diff
@@ -8,18 +8,19 @@ enum {
 	MTIME_MISSING = -2,
 };
 
+/* corresponds to a file */
```

Comment:
phony targets are `technically` files, but are never built (or never rebuilt), because phony is a no op.

Reply:
should I keep it or leave it? Also, can I add a `./vscode`, or at least add it to `.gitignore`?

Reply:
up to you. should I open an issue before making pull requests?

Reply:
besides adding a `.vscode` to `.gitignore`, I don't think this pull request has a meaningful purpose any more, since @Duncaen  already fixed the old comment.